modules/microshift-oc-mirror-list-ops-catalogs.adoc

```diff
@@ -1,54 +1,54 @@
 //Module included in the following assemblies:
 //
-//* microshift_running_apps/microshift_operators/microshift-operators-olm.adoc
+//* microshift_running_apps/microshift_operators/microshift-operators-oc-mirror.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="microshift-oc-mirror-list-operators-catalogs_{context}"]
 = Inspecting catalog contents by using the oc-mirror plugin
 
-Use the following example procedure to select a catalog and list Operators from available {OCP} content to add to your oc-mirror plugin image set configuration file.
+Use the following example procedure to select a catalog and list {OCP} Operators to add to your oc-mirror plugin image set configuration file. Selecting a catalog and listing Operators can only be done using oc mirror v1.
 
 [NOTE]
 ====
 If you use your own catalogs and Operators, you can push the images directly to your internal registry.
 ====
 
 .Prerequisites
-* The {oc-first} is installed.
-* Operator Lifecycle Manager (OLM) is installed.
-* The oc-mirror OpenShift CLI (oc) plugin is installed.
+* You uninstalled {oc-first}.
+* You installed the Operator Lifecycle Manager (OLM).
+* You installed the oc-mirror plugin.
 
 .Procedure
-. Get a list of available Red{nbsp}Hat-provided Operator catalogs to filter by running the following command:
+. Get a list of available Red Hat-provided Operator catalogs to filter by running the following command:
```

Comment:
Something to potentially follow up on in a future PR: I thought we were including {nbsp} between "Red" and "Hat".

Reply:
Yeah, toolx said no to these, then that they were ok. (180)